### PR TITLE
feature/bikes-card

### PR DESCRIPTION
**Pull request by prostoleo**
add: add bikesCard component, update tailwind config with colors for progress bar

### CODE REVIEW

File: astro/src/components/cards/BikesCard.astro [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+import Carousel from "../Carousel.astro";
```
Comment (darioguarascio) [inline astro/src/components/cards/BikesCard.astro]: Remember the @ notation

File: astro/src/components/cards/BikesCard.astro [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+const { lang, matchScore, brand, model, date, location } =
+  Astro.props as BikesCardProps;
+changeLanguage(lang);
+---
+
+<li class="link-card list-none p-4 pb-6 rounded-lg bg-white shadow-md">
+  <div>
+    <Carousel />
```
Comment (darioguarascio) [inline astro/src/components/cards/BikesCard.astro]: Carousel should receive a list of images as prop

File: astro/src/components/sections/BikesSection.astro [hunk 3]
```diff
@@ -20,5 +21,21 @@ import BaseContainer from "@components/base/BaseContainer.astro";
...
 
     <SearchBikes lang="en" />
+
+    <div
+      class="mt-4 md:mt-10 grid gap-4 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4"
+    >
+      {
+        Array.from(Array(10).keys()).map(() => (
+          <BikesCard
+            date="28/05/2022"
```
Comment (darioguarascio) [inline astro/src/components/sections/BikesSection.astro]: About date: the input valve is in form YYYY-mm-dd, and in output should be formatted using the proper, language based, format, and also include a relative notation ( x days ago)
Moment js is a library i know to perform these kinds of operations

**Review by darioguarascio — CHANGES_REQUESTED** [review]

File: astro/src/components/sections/BikesSection.astro (hunk 3 above)
Comment (prostoleo) [reply astro/src/components/sections/BikesSection.astro]: I've decided not to use moment.js as it is fairly heavy dependency. Instead I use Intl.RelativeTimeFormat.

**Review by darioguarascio — APPROVED** [review]

**Pull request merged**